### PR TITLE
Add docker-build proxy setting example

### PR DESCRIPTION
**Pull request by astronaut0131**
1. Add docker-build proxy setting in document.
2. remove `-q` option of `wget`, which makes it silent and hard to figure out what happens when there is network issue.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -78,6 +78,9 @@ CTK_TAG ?= 91ee4968b7b97996f8c466a3ebbdce41168118e3
 # additional arguments to pass to 'docker build'
 BUILD_ARGS ?=
 BUILD_ARGS := $(BUILD_ARGS) --build-arg CTK_TAG=${CTK_TAG}
+BUILD_ARGS := $(BUILD_ARGS) --build-arg http_proxy=${http_proxy}
```
Comment (avalluri) [inline Makefile:81]: @astronaut0131 The BUILD_ARGS make variable is defined to serve this purpose only. So, one can call:

`make docker-build BUILD_ARGS="--build-args http_proxy=someproxy --build-args no_proxy=skipproxy,list"`

**Review by avalluri — CHANGES_REQUESTED** [review]
@astronaut0131 Thank you so much for the contribution.  Can you resubmit that PR by removing the proxy-related change?

File: Makefile (hunk 1 above)
Comment (avalluri) [reply Makefile]: > Good idea, I think proxy setting can be mentioned in the doc, I've wasted some time on it actually.

Yes, I agree. Would be useful to document all the supported make variables.

File: README.md [hunk 2]
```diff
@@ -75,7 +75,8 @@ $ export REGISTRY="localhost:5000" # docker registry to push the container image
 $ make docker-build
 $ make docker-push
 ```
-
+> **NOTE**: If you operate behind a corporate proxy, proxy settings can be passed in like \
+```make docker-build BUILD_ARGS="--build-args http_proxy=someproxy --build-args https_proxy=someproxy --build-args no_proxy=some_list"```
```
Comment (avalluri) [inline README.md:79]: Small nit: `--build-args` -> `--build-arg`

Comment (astronaut0131) [reply README.md]: > Small nit: `--build-args` -> `--build-arg`

fixed.

**Review by avalluri — APPROVED** [review]
Looks good to me.

**Pull request merged**